gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/EntityTagsController.java

```diff
@@ -125,4 +125,20 @@ public Map post(@RequestParam("entityId") String entityId,
     return taskService.create(operation);
   }
 
+  @RequestMapping(value = "/batchUpdate", method = RequestMethod.POST)
```

Comment:
I'd probably still lean towards `/batch/tags` rather than overloading the meaning of `/tags/batch` (as we already have endpoints that work on `/tags/**`). This would probably be most cleanly implemented as a new `BatchEntityTagsController`.

Consider that a personal preference of mine ... you're free to make a choice that suits you.